src/middlewares/tokenVerify.js

```diff
@@ -0,0 +1,30 @@
+import jwt from 'jsonwebtoken';
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+const keyconfig = process.env.SECRET;
+
+const Verify = {
+    async tokenVerify (req, res, next) {
+        const token = req.headers['x-access-token'];
```

Comment:
This is fine. But you should consider a case where the user decides to pass the token as a query or use `Authorization` header instead of `x-access-token`.